mesosutil/constants.go

```diff
@@ -2,5 +2,5 @@ package mesosutil
 
 const (
 	// MesosVersion indicates the supported mesos version.
-	MesosVersion = "0.23.0"
```

Comment:
MaxMesosVersion would make sense. But mutil.MesosVersion to be potentially a different version than the installed Mesos master is strange.